src/main.py

```diff
@@ -342,18 +342,12 @@ async def main() -> None:
                             )
                             knative_serving_url = lsstcomcamsim_knative_serving_url
                             in_process_requests_gauge = lsstcomcamsim_in_process_requests_gauge
+                        case "LSSTComCam":
+                            logging.info(f"Ignore LSSTComCam message {next_visit_message_updated}.")
```

Comment:
Maybe explicitly say that there's no service for this case?

Reply:
Should it be `next_visit_message_updated`, or `next_visit_message_initial` like the previous logs?

Reply:
Looks to me that `next_visit_message_updated` is a bit better as at this point `next_visit_message_updated` exists so probably everything can just use that instead of `next_visit_message_initial`.  The previous logs using `next_visit_message_initial` are before `next_visit_message_updated` exists.